android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -235,6 +237,27 @@ public void onError(@NonNull RegistrationException error,
         };
     }
 
+    private UnregistrationListener unregistrationListener() {   
+        return new UnregistrationListener() {
+            @Override
+            public void onUnregistered(String accessToken, String fcmToken) {
+                if (BuildConfig.DEBUG) {
+                    Log.d(TAG, "Successfully unregistered FCM");
+                }
+                // eventManager.sendEvent(EVENT_DEVICE_UNREGISTERED, null);
```

Comment:
do we need to send any event to JS? Perhaps we don't need to communicate anything. If we do need to send an event we should not reuse the `EVENT_DEVICE_NOT_READY` constant

Reply:
argre with you, done